packages/main/src/Form.ts

```diff
@@ -301,6 +309,17 @@ class Form extends UI5Element {
 		});
 	}
 
+	setFastNavGroups() {
+		if (!this.hasGroupItems) {
+			this.setAttribute("data-sap-ui-fastnavgroup", "true");
+			return;
+		}
+
+		this.items.forEach((item: IFormItem) => {
```

Comment:
this could be moved to customElement decorator of ui5-form-group

Reply:
fixed